docs/advanced/document-store/weaviate.md

```diff
@@ -168,3 +168,93 @@ print(results[0].text)
 ```text
 Persist Documents with Weaviate.
 ```
+
+
+## Search with filter
+
+Search with `.find` can be restricted by user-defined filters. Such filters that can be constructed using the following operators:
+
+| Name              | Description            | Equivalent Python operator |
```

Comment:
I would link to some `weaviate` documentation for their own syntax, so that we do not need to mantain these lists

Reply:
Note that this is their own operators , not a translation maintained by us